acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -7,7 +7,7 @@ module Puppet
   module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
-        :redhat        => /fedora|el|centos/,
+        :redhat        => /fedora|el-|centos/,
```

Comment:
yep, but I don't want it to be a problem in the future, if we happen to add another OS whose name (but not version) contains `el`